changelogs/fragments/459-add-new-message-type.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - sonic-logging - Add new 'message_type' choice to the logging resource module (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/459)
```

Please describe the new choice here and add the required '.' at the end of the fragment:

```suggestion
  - sonic-logging - Add new 'auditd_system' choice to the 'message_type' choices for the logging resource module (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/459).
```